### PR TITLE
add login theme

### PR DESCRIPTION
**Pull request by LEHOANGGLAM**
- Mount folder themes in keycloak container to yas folder
- Add Yas login theme in keycloak
image

image

### CODE REVIEW

File: themes/yas/login/login.ftl [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+                    <div>
+                        <label class="visibility" id="v" onclick="togglePassword()"><img id="vi" src="${url.resourcesPath}/img/eye-off.png"></label>
+                    </div>
+                <input id="password" class="login-field" placeholder="${msg("password")}" type="password" name="password" tabindex="2">
+                <input class="submit" type="submit" value="${msg("doLogIn")}" tabindex="3">
+                </form>
+            </div>
+          
+        </#if>
+
```
Comment (thiennn) [inline themes/yas/login/login.ftl:44]: Blank lines should be removed

**Comment by thiennn** [comment]
the themes folder should be move into /identity because it's specific for keycloak

File: themes/yas/login/login.ftl (hunk 1 above)
Comment (LEHOANGGLAM) [reply themes/yas/login/login.ftl:44]: I'll update, thanks

File: themes/yas/login/resources/img/yaslogo.pngZone.Identifier [hunk 2]
```diff
@@ -0,0 +1,4 @@
+[ZoneTransfer]
```
Comment (toannguyenk) [inline themes/yas/login/resources/img/yaslogo.pngZone.Identifier:1]: the name of this file is very strange. Can you rename it ?

Comment (LEHOANGGLAM) [reply themes/yas/login/resources/img/yaslogo.pngZone.Identifier:1]: I dont what is this file too. It's auto generate when i add logo img into folder, i will check again.

**Comment by LEHOANGGLAM** [comment]
Dear @toannguyenk , @thiennn i already updated, please approve this PR for me, thanks

**Comment by LEHOANGGLAM** [comment]
Dear @toannguyenk , @thiennn i already updated, please approve this PR for me, thanks

**Review by toannguyenk — APPROVED** [review]

**Review by VoNhu2901 — APPROVED** [review]

**Pull request merged**